runtime/src/main/kotlin/kotlin/reflect/AssociatedObjects.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package kotlin.reflect
+
+import kotlin.native.internal.*
+
+/**
+ * This annotation marks the associated objects API that is considered experimental and is not subject to the
+ * [general compatibility guarantees](https://kotlinlang.org/docs/reference/evolution/components-stability.html) given for the standard library:
+ * the behavior of such API may be changed or the API may be removed completely in any further release.
+ *
+ * > Beware using the annotated API especially if you're developing a library, since your library might become binary incompatible
+ * with the future versions of the standard library.
+ *
+ * Any usage of a declaration annotated with `@ExperimentalAssociatedObjects` must be accepted either by
+ * annotating that usage with the [UseExperimental] annotation, e.g. `@UseExperimental(ExperimentalAssociatedObjects::class)`,
+ * or by using the compiler argument `-Xuse-experimental=kotlin.reflect.ExperimentalAssociatedObjects`.
+ */
```

Comment:
Better to copy-paste docs from another experimental annotation, e.g. `ExperimentalTypeInference`.

Reply:
> Better to copy-paste docs from another experimental annotation

Exactly! That's why I've copied this from [`ExperimentalStdlibApi`](https://github.com/JetBrains/kotlin/blob/43bf962d790a532981020b729d94ba3da87a316f/libraries/stdlib/src/kotlin/annotations/ExperimentalStdlibApi.kt#L6).

> e.g. `ExperimentalTypeInference`.

Do you find it better source?

Reply:
I mean not from `ExperimentalStdlibApi` but from another one, for example from `ExperimentalTypeInference`

Reply:
Done.